src/Autocomplete.soy

```diff
@@ -2,6 +2,7 @@
 
 /**
  * This renders the main element.
+ * @param? id
```

Comment:
Yeah, I see your point. It still adds a wrapper around it with some css classes though, so it would be expected that this would receive the id, instead of the next element.